fractal_server/app/runner/_common.py

```diff
@@ -480,6 +480,15 @@ def call_parallel_task(
     for _ in map_iter:
         pass  # noqa: 701
 
+    pattern = "*_*.metadiff.json"
```

Comment:
1. We would use the server-side directory, and not user-side.
2. Maybe we can avoid globbing, since we already know component list and TaskFiles encodes all sort of prefix/suffix.

Reply:
3. Should we also modify the output of `call_single_parallel_task` so that it returns the metadiff dictionary? This will avoid a lot of file reading.